packages/db/tests/utls.ts

```diff
@@ -271,6 +271,7 @@ export function mockSyncCollectionOptionsNoInitialState<
   let write: Parameters<SyncConfig<T>[`sync`]>[0][`write`]
   let commit: () => void
   let markReady: () => void
+  let truncate: () => void
```

Comment:
Can we fix the name of this file please? It contains a typo. Should be `utils.ts`